datadog_checks_base/tests/test_time.py

```diff
@@ -0,0 +1,60 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datetime import datetime
+
+import pytest
+from dateutil import tz
+from six import PY2
+
+from datadog_checks.base.utils.time import EPOCH, UTC, get_current_datetime, get_timestamp, normalize_datetime
+
+pytestmark = pytest.mark.time
+
+
+def test_timestamp_type():
+    assert isinstance(get_timestamp(), float)
+
+
+class TestNormalization:
+    def test_replace(self):
+        now = datetime.now()
+        assert now.tzinfo is None
+
+        assert normalize_datetime(now).tzinfo is UTC
+
+    def test_utc(self):
+        nyc = tz.gettz('America/New_York')
+        now = datetime.now(nyc)
+
+        assert normalize_datetime(now).tzinfo is nyc
```

Comment:
Do you also want to assert `normalize_datetime(now) is now`? Creating a new datetime can be annoyingly expensive considering these things are immutable 😛.

Reply:
Done, thanks!